src/hooks/action.test.ts

```diff
@@ -166,4 +174,46 @@ describe('Action Hooks', () => {
       expect(queryClient.getQueryData(key)).toBeFalsy();
     });
   });
+
+  describe('useMemberActions', () => {
+    const args = {
+      startDate: '2024-04-24T14:07:00.074Z',
+      endDate: '2024-04-24T14:07:00.074Z',
+    };
+    const route = `/${buildGetMemberActionsRoute(args)}`;
+    const key = memberKeys.current().actions(args);
+
+    const response = ACTIONS_DATA;
+
+    it(`Receive actions for current member`, async () => {
+      const hook = () => hooks.useMemberActions(args);
```

Comment:
Do you need to redefine the hook in each test ? can it be defined in the describe instead ?